box.py

```diff
@@ -0,0 +1,47 @@
+import numpy as np
+
+class Box:
+    def __init__(self, box_dim):
+        self.box_dim = box_dim
+        
+    # wrapping equation
+    def wrap(self, box_dim, coordinates):
```

Comment:
I think you can remove the box_dim argument, is not been used. Intead you're using 'self.box_dim', I thinks is better.

Reply:
addressed